generators/app/templates/package.json

```diff
@@ -4,10 +4,10 @@
     "@fortawesome/fontawesome-svg-core": "^1.2.12",
     "@fortawesome/free-solid-svg-icons": "^5.6.3",
     "@fortawesome/react-fontawesome": "^0.1.4",
-    "@inrupt/inrupt-atomic-style-guide": "^0.1.19",
-    "@inrupt/solid-react-components": "^0.2.2",
+    "@inrupt/solid-style-guide": "^0.1.21",
+    "@inrupt/solid-react-components": "0.3.0-rc.0",
```

Comment:
We can make this 0.3.0 now since it is published

Reply:
done